terraform/environments/prd/aws/main.tf

```diff
@@ -27,3 +27,25 @@ module "vpcs" {
   nat_gws = each.value.nat_gws
   subnets = each.value.subnets
 }
+
+
+
+module "gigachad_rds" {
+  source = "../../../modules/aws/rds/"
+  for_each = local.rdss
+  vpc_id = module.vpcs[each.value.gigachad_rds.name].vpc_id
```

Comment:
if we use attribute `each.value.gigachad_rds.name` it should probably have name vpc_name